client/common.go

```diff
@@ -199,21 +199,24 @@ func (x *contextCall) processResponse() bool {
 	// get result status
 	st := apistatus.FromStatusV2(x.resp.GetMetaHeader().GetStatus())
 
-	successfulStatus := apistatus.IsSuccessful(st)
-	x.err = apistatus.ErrFromStatus(st)
+	var isErrorExists bool
+	x.err, isErrorExists = st.(error)
```

Comment:
Combination of `IsSuccessful` and `ErrFromStatus` to the same, under the hood, but in two different operations.